core/terraform/servicebus.tf

```diff
@@ -6,6 +6,24 @@ resource "azurerm_servicebus_namespace" "sb" {
   capacity            = "1"
   tags                = local.tre_core_tags
 
+  # Block public access
```

Comment:
Has this been tested on an existing environment?
Switching the way inside TF we accomplish the same Azure setup has caused issues in the past (required a migration or other mitigation).

Reply:
Hmm, good point, no it hasn't. I'll try give it a test.

Reply:
Seems like some discussion here - https://github.com/hashicorp/terraform-provider-azurerm/issues/23954

Might need to add:

`terraform state rm azurerm_servicebus_namespace_network_rule_set.servicebus_network_rule_set`

to the `migrate.sh`.

Reply:
I've added the migration step. Deployed from main, then from this branch. All good, can see it being removed from state in the deployment step.